src/libsystemd/sd-device/device-enumerator.c

```diff
@@ -33,6 +33,7 @@ struct sd_device_enumerator {
         size_t n_devices, current_device_index;
         bool scan_uptodate;
         bool sorted;
+        bool match_always_trigger_parent;
```

Comment:
Please move this to the end of the struct, and name something like the following:
`MatchFlag parent_match_flags;`
Of course, the definition of `MatchFlag` enum needs to be moved above.

Then, initialize it in `sd_device_enumerator_new()` with `MATCH_ALL`.